include/json/config.h

```diff
@@ -12,6 +12,7 @@
 #include <ostream>
 #include <sstream>
 #include <string>
+#include <string_view>
```

Comment:
It seems the system doesn't support C++17 Standard now, so it cannot open include file `string_view`
but if you like, you could use the __plusplus marco switch : 
```
#if __cplusplus >= 201703L 
#include <string_view>
#endif 
```